### DISCUSSION TITLE
rocketmq删除topic，导致broker挂掉

### DISCUSSION THREAD

**Discussion by z499738316** [apache/rocketmq#7973] [Q&A]
有一个topic因为java服务的原因，堆积了很多的消息，然后我们在删除这个topic后，broker就突然挂掉了，这个是什么原因导致的啊，为啥删除topic会导致broker挂掉

**Comment by leizhiyuan** [comment]
怎么操作的删除？

另外可以提供一下删除那段时间的broker日志

**Comment by z499738316** [comment]
> 怎么操作的删除？
> 
> 另外可以提供一下删除那段时间的broker日志
RocketMQ-Dashboard用这个删除的，日志估计找不到，反正用这个面板删除后broker挂掉了，搭的主从 一主俩从
